sqldelight-compiler/integration-tests/src/test/kotlin/com/example/PlayerQueries.kt

```diff
@@ -110,7 +110,7 @@ class PlayerQueries(private val database: TestDatabase, private val driver: SqlD
         val numberIndexes = createArguments(count = number.size, offset = 2)
         driver.execute(null, """
         |UPDATE player
-        |SET team = ?1
+        |SET [team] = ?1
```

Comment:
Let me know if you want to revert this, I mostly used it to sanity check my implementation. I could also add separate tests of course.

Reply:
A dedicated test would be nice because right now because nothing prevents the integration test from changing completely in the future and ignoring the need to cover this case.

Reply:
Done